docs/how-to-guides/integrating-autoware/tuning-parameters-and-performance/important-temporal-performance-metrics.md

```diff
@@ -0,0 +1,121 @@
+| APER-021 | **Response time** from CPP #6 to CPP #17 via CPP #12 | Response time to consume data sent from DNN-based Object Recognition after LiDARs output pointcloud. | Object Recognition        | N/A             | Validator relies on fresh and update-date data for data synchronization.                                            |                                                                                        |
+
+### Detailed temporal performance metrics for Paths between Obstacle segmentation and Planning
+
+Obstacle segmentation, which is a crucial part of Perception, transmits data to Planning. The figure below illustrates the model that takes into account performance metrics related to Obstacle segmentation and Planning.
+
+![Model for Obstacle segmentation temporal performance metrics](./images/important-temporal-performance-metrics/model-for-obstacle-segmentation-metrics.svg)
+
+!!! note
+
+    Both the Obstacle grid map and Obstacle segmentation transmit data to multiple sub-components of Planning. However, not all of these sub-components are described in the model. This is because our primary focus is on the paths from LiDAR to Planning via Obstacle segmentation.
+
+The following list shows the temporal performance metrics around Obstacle segmentation and Planning.
+
+| ID       | Representation in the model                          | Metric meaning                                                                           | Related functionality | Reference value | Reason to choose it as a metric                                                                                      | Note |
+| -------- | ---------------------------------------------------- | ---------------------------------------------------------------------------------------- | --------------------- | --------------- | -------------------------------------------------------------------------------------------------------------------- | ---- |
+| OSEG-001 | **Message rate** from CPS #4 to CPS #7               | Update rate of Occupancy grid map received by Planning (`behavior_path_planner`)         | Obstacle segmentation | 10 Hz           | Planning relies on occupancy grid map to be updated frequently and smoothly for creating accurate trajectory.        |      |
+| OSEG-002 | **Response time** from CPS #0 to CPS #9 via CPS #7   | Response time to consume Occupancy grid map after LiDARs output sensing data.            | Obstacle segmentation | N/A             | Planning relies on fresh and up-to-date perceived data from occupancy grid map for creating accurate trajectory..    |      |
+| OSEG-003 | **Message rate** from CPS #6 to CPS #11              | Update rate of obstacle segmentation received by Planning (`behavior_velocity_planner`). | Obstacle segmentation | 10 Hz           | Obstacle segmentation must be updated frequently and smoothly for creating accurate trajectory..                     |      |
```

Comment:
It would be better to use the same sentence format as OSEG-001 like: "Planning relies on ..."